packages/core/src/external/aws/app-config.ts

```diff
@@ -79,6 +86,34 @@ export async function getFeatureFlags(
   throw new MetriportError(`Failed to get Feature Flags`);
 }
 
+export async function getFeatureFlagValueStringArrayCxValues<
+  T extends keyof StringValueFeatureFlags
+>(
+  region: string,
+  appId: string,
+  configId: string,
+  envName: string,
+  cxId: string,
+  featureFlagNames?: T[]
+): Promise<CxFeatureFlagStatus> {
+  const configContentValue = await getFeatureFlags(region, appId, configId, envName);
+  const targetFeatureFlags =
+    featureFlagNames && featureFlagNames.length > 0
+      ? featureFlagNames
+      : stringValueFFsSchema.keyof().options;
```

Comment:
I think we should have a const w/ the names of cx-based FFs. Right now this is including all string-based FFs, including `oidsWithIHEGatewayV2Enabled` that contains a lot of unrelated external GW OIDs and the E2E cx ids.

I was validating this was doable and ended up coding it, if you want to steal it:

```ts
export const cxBasedFFsSchema = z.object({
  cxsWithEnhancedCoverageFeatureFlag: ffStringValuesSchema,
  cxsWithCQDirectFeatureFlag: ffStringValuesSchema,
  cxsWithCWFeatureFlag: ffStringValuesSchema,
  cxsWithADHDMRFeatureFlag: ffStringValuesSchema,
  cxsWithNoWebhookPongFeatureFlag: ffStringValuesSchema,
  cxsWithIncreasedSandboxLimitFeatureFlag: ffStringValuesSchema,
  cxsWithIHEGatewayV2Enabled: ffStringValuesSchema,
  cxsWithEpicEnabled: ffStringValuesSchema,
});
export type CxBasedFFsSchema = z.infer<typeof cxBasedFFsSchema>;

export const stringValueFFsSchema = cxBasedFFsSchema.merge(
  z.object({
    oidsWithIHEGatewayV2Enabled: ffStringValuesSchema,
    e2eCxIds: ffStringValuesSchema.nullish(),
  })
);
export type StringValueFeatureFlags = z.infer<typeof stringValueFFsSchema>;

export type CxFeatureFlagStatus = Partial<Record<keyof CxBasedFFsSchema, boolean>>;
```

and update this

```ts
export async function getFeatureFlagValueStringArrayCxValues<T extends keyof CxBasedFFsSchema>(
...
```

Reply:
Thank you for the code -- done!